aws/data_source_aws_msk_cluster_test.go

```diff
@@ -22,8 +23,8 @@ func TestAccAWSMskClusterDataSource_Name(t *testing.T) {
 				Config: testAccMskClusterDataSourceConfigName(rName),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttrPair(resourceName, "arn", dataSourceName, "arn"),
-					resource.TestCheckResourceAttrSet(dataSourceName, "bootstrap_brokers"),
-					resource.TestCheckResourceAttrSet(dataSourceName, "bootstrap_brokers_tls"),
+					resource.TestCheckResourceAttr(resourceName, "bootstrap_brokers", ""),
+					resource.TestMatchResourceAttr(resourceName, "bootstrap_brokers_tls", regexp.MustCompile(`^(([-\w]+\.){1,}[\w]+:\d+,){2,}([-\w]+\.){1,}[\w]+:\d+$`)), // Ordering not guaranteed
```

Comment:
could we use `TestCheckResourceAttrPair` checks in exchange of these checks on the resource?

Reply:
Ah sorry the Go comment wasn't clear, unfortunately the ordering of the hostnames within the string response isn't guaranteed between read calls, so the resource and the data source may have differing responses. This resource attribute may be a candidate in the future for us going against the API and making a `TypeSet` for better usability.

Reply:
Ohhh gotcha, makes sense!